packages/jsii-rosetta/test/translations/imports/submodule-import.cs

```diff
@@ -9,7 +9,7 @@
 });
 
 // Accesses two distinct points of the submodule hierarchy
-var myClass = new Submodule.MyClass(new SomeStruct { Prop = Submodule.Child.SomeEnum.SOME });
+var myClass = new Submodule.MyClass(new SomeStruct { Prop = Submodule.Child.SomeEnum.SOME, NestedStruct = new Submodule.Child.AnotherStruct { StringProperty = "hello" } });
```

Comment:
This currently fails as the result from Rosetta is missing the `Submodule.Child.` for `AnotherStruct`
<img width="960" alt="image" src="https://user-images.githubusercontent.com/1112056/231725845-ee78e038-862f-4338-afd0-4e861ea244d6.png">

Reply:
While looking at this once more, I noticed that the original test already seems to have a small error, too:
```suggestion
var myClass = new Submodule.MyClass(new Submodule.Child.SomeStruct { Prop = Submodule.Child.SomeEnum.SOME, NestedStruct = new Submodule.Child.AnotherStruct { StringProperty = "hello" } });
```